web/src/ApiButton.tsx

```diff
@@ -279,20 +307,20 @@ function ApiButtonWithOptions(props: ApiButtonWithOptionsProps & ButtonProps) {
           setOpen(false)
         }}
         anchorEl={anchorRef.current}
-        title={`Options for ${props.uiButton.spec?.text}`}
+        title={`Options for ${text}`}
       >
         <ApiButtonForm {...props} />
       </FloatDialog>
     </>
   )
 }
 
-export const ApiIcon = (props: ApiIconProps) => {
-  if (props.iconSVG) {
+export const ApiIcon = ({ iconName, iconSVG }: ApiIconProps) => {
```

Comment:
I always forget about all the fun destructuring things you can do in modern JS ✨